promptsource/templates/swag/regular/templates.yaml

```diff
@@ -2,240 +2,130 @@ dataset: swag
-    answer_choices: null
-    answer_choices_key: null
-    id: a453f563-16d0-4021-bc00-a3806e70c589
-    jinja: 'Consider the sentence: {{ sent1 }}
-
-      Is the following an appropriate continuation?
-
-      {{ sent2 }} {{ [ending0, ending1, ending2, ending3][1] }}
-
-      Yes or No?
-
-      ||| {{ {0: "No", 1: "Yes", 2: "No", 3: "No"}[label] }}'
+    reference: Generate the ending
+  d61be86a-242e-48ad-871c-f8be5133c4df: !Template
+    answer_choices:
+    - (a)
+    - (b)
+    - (c)
+    - (d)
+    answer_choices_key: ending0 ||| ending1 ||| ending2 ||| ending3
+    id: d61be86a-242e-48ad-871c-f8be5133c4df
+    jinja: "First, {{ sent1.lower() }} Then, {{ sent2.lower() }}... \nChoose the key\
+      \ with an appropriate ending:\n(a) {{ ending0 }}\n(b) {{ ending1 }}\n(c) {{\
+      \ ending2 }}\n(d) {{ ending3 }}\n||| \n{{answer_choices[label]}}\n"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_1
-    reference: ''
-  b0e9c3bf-7791-41fd-bb8d-a55dfd91cb9c: !Template
+      metrics:
+      - Accuracy
+      original_task: true
+    name: first_then_key
+    reference: Predict the key of the ending
+  dae2999a-843e-445f-819e-9a3255cca049: !Template
     answer_choices: null
-    answer_choices_key: null
-    id: b0e9c3bf-7791-41fd-bb8d-a55dfd91cb9c
-    jinja: '{{ startphrase }}...
-
-      Given this beginning, what is the appropriate ending?
-
-      Ending 1: {{ ending0 }}
-
-      Ending 2: {{ ending1 }}
-
-      Ending 3: {{ ending2 }}
-
-      Ending 4: {{ ending3 }}
-
-
-      ||| {{ {0: "Ending 1", 1: "Ending 2", 2: "Ending 3", 3: "Ending 4"}[label] }}'
+    answer_choices_key: sent1
+    id: dae2999a-843e-445f-819e-9a3255cca049
+    jinja: "{% set endings = [ending0, ending1, ending2, ending3] %}\nGenerate the\
+      \ starting sentence with the ending: {{endings[label]}}\n||| \n{{sent1}}"
     metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
       original_task: false
-    name: which_ending
-    reference: Given the beginning choose the appropriate ending
-  d61be86a-242e-48ad-871c-f8be5133c4df: !Template
+    name: generate_start
+    reference: Template generates the start of the sentence
+  ecb7eddb-5836-4b31-89d6-e2d4ebfcc779: !Template
     answer_choices: null
-    answer_choices_key: null
-    id: d61be86a-242e-48ad-871c-f8be5133c4df
-    jinja: "First, {{ sent1.lower() }} Then, {{ sent2.lower() }}... \nComplete with\
-      \ an appropriate ending:\nEnding 1: {{ ending0 }}\nEnding 2: {{ ending1 }}\n\
-      Ending 3: {{ ending2 }}\nEnding 4: {{ ending3 }}\n\n||| {{ {0: \"Ending 1\"\
-      , 1: \"Ending 2\", 2: \"Ending 3\", 3: \"Ending 4\"}[label] }}"
+    answer_choices_key: ending0 ||| ending1 ||| ending2 ||| ending3
```

Comment:
Same as the other one, please wrap in curly braces and use answer_choices in the template to catch any issues.